common/app/model/Cached.scala

```diff
@@ -146,6 +145,8 @@ object NoCache {
 
 case class NoCache[A](action: Action[A]) extends Action[A] {
 
+  implicit val ec: ExecutionContext = ExecutionContexts.executionContext
```

Comment:
If I understand correctly, because it's overriding a method, it can't change the signature